README.md

```diff
@@ -1,7 +1,27 @@
-# HoloRepository
-The COMP0111 project MSGOSHHOLO, where we develop a software engineering solution for Microsoft and GOSH.
+# HoloRepository-HoloLens
+The HoloRepository is a system for transforming medical imaging studies such as CT or MRI scans into 3-dimensional holograms, storing data on a cloud-based platform and making it available for other systems. This repository is dedicated to the portions of the system that are to be deployed on the Microsoft HoloLens:
+
+* **HoloStorageConnector**: A Unity library handling the runtime network connections from HoloLens applications to the HoloStorage. Distributed as a separate UnityPackage, this is meant to facilitate development of 3rd party applications that plug into the HoloRepository ecosystem.
+* **HoloRepository demo application**: A simple application that demonstrates how to dynamically access 3D models stored in the HoloStorage. The scenes can be distributed alongside the Connector library and serve as examples and interactive documentation.
+
+
+## Description
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/11090412/62010805-49d5b180-b167-11e9-8f36-af05454f3a15.png" alt="screenshot" height="350" />
+</p>
+
+To facilitate the access to the HoloStorage for 3rd party projects and future projects, we provide a library which is distributed as AssetPackage, can be imported into Unity projects and added to Scenes as a configurable GameObject. It provides a layer of abstraction for the network calls and API accesses performed to dynamically load data from the HoloStorage into HoloLens applications at runtime.
+
+To demonstrate the capabilities of the HoloRepository system, it is crucial to have an actual HoloLens application that puts together the pieces (and thereby also acts as an integration test). As an additional benefit, this demo application could be shipped with the Connector library as example scenes.
+
+The demo application will use the HoloStorageConnector to access data stored in the HoloStorage. The data in turn will have been generated through the HoloPipelines workflow or directly uploaded from the HoloRepository. The application will provide a basic UI and list the 3D models stored in the HoloStorage. They can then be loaded and placed, scaled and rotated within the augmented reality.
+
+To our best knowledge, there have been no projects that combine medical holograms and the FHIR standard. This project is the first to close that gap.
+
+
+## Development
+
```

Comment:
I noticed that the first row in the table is empty, is this something we can fix or is this just how GitHub renders tables?